sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java

```diff
@@ -52,6 +52,7 @@
 		new FeatureType[] {
 			TASK_MANAGEMENT }),
 	MANUAL_EXTERNAL_MESSAGES(true, true, null),
+	CONFIRMED_CASE_NOTIFICATIONS(true, true, null),
```

Comment:
Rename to EVENT_PARTICIPANT_CASE_CONFIRMED_NOTIFICATIONS (very long, I know, but the current name sounds like it's associated to ALL notifications sent out when a case gets confirmed)